pkg/cmd/join/exec.go

```diff
@@ -54,6 +54,11 @@ func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
 		WorkImageVersion:         versionBundle.Work,
 		OperatorImageVersion:     versionBundle.Operator,
 	}
+	klog.V(3).InfoS(
```

Comment:
> InfoS(msg string, keysAndValues ...interface{}) 

i think an empty string is needed as the first argument

Reply:
right, updated just now